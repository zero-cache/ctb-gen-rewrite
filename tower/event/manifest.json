["Anarchist.png","Scarecrow.png","Elf.png","Resting Soldier.png","Huntsman.png"]
