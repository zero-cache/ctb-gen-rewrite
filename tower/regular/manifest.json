["Scout.png","Sniper.png","Fragger.png","Soldier.png","Farm.png","Cryo-Gunner.png","Shotgunner.png","Tuber.png","Patrol.png","Barracks.png","Archer.png","Knifer.png","Mercenary.png","Aviator.png","Flamethrower.png","Commando.png","Slugger.png","Commander.png","DJ.png","Mortar.png","Plasma Trooper.png","Hunter.png","Railgunner.png","Paralyzer.png","Phaser.png","Helicopter.png","ZED.png"]
