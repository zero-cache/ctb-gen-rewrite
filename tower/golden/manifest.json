["Golden Scout.png","Golden Commando.png"]
